src/baldr/tz_alt.cpp

```diff
@@ -439,7 +429,10 @@ get_tz_dir()
 static_assert(min_year <= max_year, "Configuration error");
 #endif
 
+#if 0
 static std::unique_ptr<tzdb> init_tzdb();
+#endif
```

Comment:
yeah this is the crux of the change i made here when we took a copy of this and patched it. im open to better options here but none really occured to me at the time

Reply:
Keeping this unused code as a reminder of the "looking for better options"